### PR TITLE
Add session duration options

### PR DESCRIPTION
**Pull request by Flurbb**

### CODE REVIEW

File: README.md
Comment (cshamrick) [inline README.md:1]: Can you also update the Usage section of the readme where appropriate?

**Review by cshamrick — CHANGES_REQUESTED** [review]

**Comment by phillipjf** [comment]
This is fine with me. Just note that the upper bound on the duration is still constrained by the duration set on the role itself (or 1 hour in the case of role-chaining). See [`AssumeRole` API docs; `DurationSeconds`](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html#API_AssumeRole_RequestParameters):

> The maximum session duration setting can have a value from 1 hour to 12 hours. If you specify a value higher than this setting or the administrator setting (whichever is lower), the operation fails.

> However, if you assume a role using role chaining and provide a DurationSeconds parameter value greater than one hour, the operation fails.

**Review by phillipjf — APPROVED** [review]

**Comment by cshamrick** [comment]
> This is fine with me. Just note that the upper bound on the duration is still constrained by the duration set on the role itself (or 1 hour in the case of role-chaining). See [`AssumeRole` API docs; `DurationSeconds`](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html#API_AssumeRole_RequestParameters):
> 
> > The maximum session duration setting can have a value from 1 hour to 12 hours. If you specify a value higher than this setting or the administrator setting (whichever is lower), the operation fails.
> 
> > However, if you assume a role using role chaining and provide a DurationSeconds parameter value greater than one hour, the operation fails.

@Flurbb can you test related edge-cases and make sure `stsauth` handles related errors and prints useful information back to the end-user?

**Comment by Flurbb** [comment]
@cshamrick yeah can do

**Comment by Flurbb** [comment]
I'm just taking out the assume role option since it's not really going to get us anything valuable

**Comment by phillipjf** [comment]
> I'm just taking out the assume role option since it's not really going to get us anything valuable

It wasn't often, but I had a few use cases that I leveraged this functionality for managing assume-role permissions.

**Comment by Flurbb** [comment]
> > I'm just taking out the assume role option since it's not really going to get us anything valuable
> 
> It wasn't often, but I had a few use cases that I leveraged this functionality for managing assume-role permissions.

Oh I'm not taking out the assume role functionality, I'm taking out the option I added to change the duration of assume-role sessions to be something different than 1 hour. There's no functionality regression

**Review by cshamrick — APPROVED** [review]

**Pull request merged**